<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE TS>
<TS version="2.1" language="en">
<context>
    <name>ApplicationError</name>
    <message>
        <source>NoError:Name</source>
        <translation>Unknown Error</translation>
    </message>
    <message>
        <source>NoError:Description</source>
        <translation>Unknown error occurred</translation>
    </message>
    <message>
        <source>LocalizationError:Name</source>
        <translation>Localization Error</translation>
    </message>
    <message>
        <source>LocalizationError:Description</source>
        <translation>Unable to initialize locale
</translation>
    </message>
    <message>
        <source>UnknownError:Name</source>
        <translation>Unknown Application Error</translation>
    </message>
    <message>
        <source>UnknownError:Description</source>
        <translation>An unknown application error has occurred</translation>
    </message>
</context>
<context>
    <name>ServerError</name>
    <message>
        <source>NoError:Name</source>
        <translation>Unknown Server Error</translation>
    </message>
    <message>
        <source>NoError:Description</source>
        <translation>Unknown error occurred</translation>
    </message>
    <message>
        <source>BadRequest:Name</source>
        <translation>Invalid network request</translation>
    </message>
    <message>
        <source>BadRequest:Description</source>
        <translation>The network request was invalid</translation>
    </message>
    <message>
        <source>ResourceNotFound:Name</source>
        <translation>Resource not found</translation>
    </message>
    <message>
        <source>ResourceNotFound:Description</source>
        <translation>The requested resource was not found</translation>
    </message>
    <message>
        <source>UnknownError:Name</source>
        <translation>Unknown Server Error</translation>
    </message>
    <message>
        <source>UnknownError:Description</source>
        <translation>An unknown server error has occurred</translation>
    </message>
    <message>
        <source>InvalidRequest:Name</source>
        <translation>Invalid Request Error</translation>
    </message>
    <message>
        <source>InvalidRequest:Description</source>
        <translation>The request is missing a required parameter, includes an unsupported parameter or parameter value, or is otherwise malformed</translation>
    </message>
    <message>
        <source>InvalidClient:Name</source>
        <translation>Invalid Client Error</translation>
    </message>
    <message>
        <source>InvalidClient:Description</source>
        <translation>The client identifier provided is invalid</translation>
    </message>
    <message>
        <source>UnauthorizedClient:Name</source>
        <translation>Unauthorized Client Error</translation>
    </message>
    <message>
        <source>UnauthorizedClient:Description</source>
        <translation>The client is not authorized to use the requested response type</translation>
    </message>
    <message>
        <source>RedirectUriMismatch:Name</source>
        <translation>Redirect Uri Mismatch Error</translation>
    </message>
    <message>
        <source>RedirectUriMismatch:Description</source>
        <translation>The redirection URI provided does not match a pre-registered value</translation>
    </message>
    <message>
        <source>AccessDenied:Name</source>
        <translation>Access Denied Error</translation>
    </message>
    <message>
        <source>AccessDenied:Description</source>
        <translation>The end-user or authorization server denied the request</translation>
    </message>
    <message>
        <source>UnsupportedResponseType:Name</source>
        <translation>Unsupported Response Type Error</translation>
    </message>
    <message>
        <source>UnsupportedResponseType:Description</source>
        <translation>The requested response type is not supported by the authorization server</translation>
    </message>
    <message>
        <source>UnsupportedGrantType:Name</source>
        <translation>Unsupported Grant Type Error</translation>
    </message>
    <message>
        <source>UnsupportedGrantType:Description</source>
        <translation>The requested username or password are not recognized</translation>
    </message>
    <message>
        <source>InvalidGrant:Name</source>
        <translation>Invalid Grant Type Error</translation>
    </message>
    <message>
        <source>InvalidGrant:Description</source>
        <translation>The requested username or password are not recognized</translation>
    </message>
    <message>
        <source>InvalidScope:Name</source>
        <translation>Invalid Scope Error</translation>
    </message>
    <message>
        <source>InvalidScope:Description</source>
        <translation>The requested scope is invalid, unknown, or malformed</translation>
    </message>
    <message>
        <source>EmptyToken:Name</source>
        <translation>Empty Token Error</translation>
    </message>
    <message>
        <source>EmptyToken:Description</source>
        <translation>The access token is empty</translation>
    </message>
</context>
<context>
    <name>NetworkError</name>
    <message>
        <source>NoError:Name</source>
        <translation>Unknown Network Error</translation>
    </message>
    <message>
        <source>NoError:Description</source>
        <translation>Unknown error occurred</translation>
    </message>
    <message>
        <source>ConnectionRefusedError:Name</source>
        <translation>Connection Refused Error</translation>
    </message>
    <message>
        <source>ConnectionRefusedError:Description</source>
        <translation>The remote server refused the connection (the server is not accepting requests)</translation>
    </message>
    <message>
        <source>RemoteHostClosedError:Name</source>
        <translation>Remote Host Closed Error</translation>
    </message>
    <message>
        <source>RemoteHostClosedError:Description</source>
        <translation>The remote server closed the connection prematurely, before the entire reply was received and processed</translation>
    </message>
    <message>
        <source>HostNotFoundError:Name</source>
        <translation>Host Not Found Error</translation>
    </message>
    <message>
        <source>HostNotFoundError:Description</source>
        <translation>The remote host name was not found (invalid hostname)</translation>
    </message>
    <message>
        <source>TimeoutError:Name</source>
        <translation>Timeout Error</translation>
    </message>
    <message>
        <source>TimeoutError:Description</source>
        <translation>The connection to the remote server timed out</translation>
    </message>
    <message>
        <source>OperationCanceledError:Name</source>
        <translation>Operation Canceled Error</translation>
    </message>
    <message>
        <source>OperationCanceledError:Description</source>
        <translation>The operation was canceled via calls to abort() or close() before it was finished</translation>
    </message>
    <message>
        <source>SslHandshakeFailedError:Name</source>
        <translation>SSL Handshake Failed Error</translation>
    </message>
    <message>
        <source>SslHandshakeFailedError:Description</source>
        <translation>The operation was canceled via calls to abort or close before it was finished</translation>
    </message>
    <message>
        <source>TemporaryNetworkFailureError:Name</source>
        <translation>Temporary Network Failure Error</translation>
    </message>
    <message>
        <source>TemporaryNetworkFailureError:Description</source>
        <translation>The connection was broken due to disconnection from the network, however the system has initiated roaming to another access point. The request should be resubmitted and will be processed as soon as the connection is re-established</translation>
    </message>
    <message>
        <source>NetworkSessionFailedError:Name</source>
        <translation>Network Session Failed Error</translation>
    </message>
    <message>
        <source>NetworkSessionFailedError:Description</source>
        <translation>The connection was broken due to disconnection from the network or failure to start the network</translation>
    </message>
    <message>
        <source>BackgroundRequestNotAllowedError:Name</source>
        <translation>Background Request Not Allowed Error</translation>
    </message>
    <message>
        <source>BackgroundRequestNotAllowedError:Description</source>
        <translation>The background request is not currently allowed due to platform policy</translation>
    </message>
    <message>
        <source>ProxyConnectionRefusedError:Name</source>
        <translation>Proxy Connection Refused Error</translation>
    </message>
    <message>
        <source>ProxyConnectionRefusedError:Description</source>
        <translation>The connection to the proxy server was refused (the proxy server is not accepting requests)</translation>
    </message>
    <message>
        <source>ProxyConnectionClosedError:Name</source>
        <translation>Proxy Connection Closed Error</translation>
    </message>
    <message>
        <source>ProxyConnectionClosedError:Description</source>
        <translation>The proxy server closed the connection prematurely, before the entire reply was received and processed</translation>
    </message>
    <message>
        <source>ProxyNotFoundError:Name</source>
        <translation>Proxy Not Found Error</translation>
    </message>
    <message>
        <source>ProxyNotFoundError:Description</source>
        <translation>The proxy host name was not found (invalid proxy hostname)</translation>
    </message>
    <message>
        <source>ProxyTimeoutError:Name</source>
        <translation>Proxy Timeout Error</translation>
    </message>
    <message>
        <source>ProxyTimeoutError:Description</source>
        <translation>The connection to the proxy timed out or the proxy did not reply in time to the request sent</translation>
    </message>
    <message>
        <source>ProxyAuthenticationRequiredError:Name</source>
        <translation>Proxy Authentication Required Error</translation>
    </message>
    <message>
        <source>ProxyAuthenticationRequiredError:Description</source>
        <translation>The proxy requires authentication in order to honor the request but did not accept any credentials offered (if any)</translation>
    </message>
    <message>
        <source>ContentAccessDenied:Name</source>
        <translation>Content Access Denied Error</translation>
    </message>
    <message>
        <source>ContentAccessDenied:Description</source>
        <translation>The access to the remote content was denied (similar to HTTP error 401)</translation>
    </message>
    <message>
        <source>ContentOperationNotPermittedError:Name</source>
        <translation>Content Operation Not Permitted Error</translation>
    </message>
    <message>
        <source>ContentOperationNotPermittedError:Description</source>
        <translation>The operation requested on the remote content is not permitted</translation>
    </message>
    <message>
        <source>ContentNotFoundError:Name</source>
        <translation>Content Not Found Error</translation>
    </message>
    <message>
        <source>ContentNotFoundError:Description</source>
        <translation>The remote content was not found at the server (similar to HTTP error 404)</translation>
    </message>
    <message>
        <source>AuthenticationRequiredError:Name</source>
        <translation>Authentication Required Error</translation>
    </message>
    <message>
        <source>AuthenticationRequiredError:Description</source>
        <translation>The remote server requires authentication to serve the content but the credentials provided were not accepted (if any)</translation>
    </message>
    <message>
        <source>ContentReSendError:Name</source>
        <translation>Content Resend Error</translation>
    </message>
    <message>
        <source>ContentReSendError:Description</source>
        <translation>The request needed to be sent again, but this failed for example because the upload data could not be read a second time</translation>
    </message>
    <message>
        <source>ProtocolUnknownError:Name</source>
        <translation>Protocol Unknown Error</translation>
    </message>
    <message>
        <source>ProtocolUnknownError:Description</source>
        <translation>The Network Access API cannot honor the request because the protocol is not known</translation>
    </message>
    <message>
        <source>ProtocolInvalidOperationError:Name</source>
        <translation>Protocol Invalid Operation Error</translation>
    </message>
    <message>
        <source>ProtocolInvalidOperationError:Description</source>
        <translation>The requested operation is invalid for this protocol</translation>
    </message>
    <message>
        <source>UnknownNetworkError:Name</source>
        <translation>Unknown Network Error</translation>
    </message>
    <message>
        <source>UnknownNetworkError:Description</source>
        <translation>An unknown network-related error was detected</translation>
    </message>
    <message>
        <source>UnknownProxyError:Name</source>
        <translation>Unknown Proxy Error</translation>
    </message>
    <message>
        <source>UnknownProxyError:Description</source>
        <translation>An unknown proxy-related error was detected</translation>
    </message>
    <message>
        <source>UnknownContentError:Name</source>
        <translation>Unknown Content Error</translation>
    </message>
    <message>
        <source>UnknownContentError:Description</source>
        <translation>An unknown error related to the remote content was detected</translation>
    </message>
    <message>
        <source>ProtocolFailure:Name</source>
        <translation>Protocol Failure Error</translation>
    </message>
    <message>
        <source>ProtocolFailure:Description</source>
        <translation>A breakdown in protocol was detected (parsing error, invalid or unexpected responses, etc.)</translation>
    </message>
    <message>
        <source>UnknownError:Name</source>
        <translation>Unknown Network Error</translation>
    </message>
    <message>
        <source>UnknownError:Description</source>
        <translation>An unknown network error has occurred</translation>
    </message>
</context>
<context>
    <name>SSLNetworkError</name>
    <message>
        <source>NoError:Name</source>
        <translation>Unknown SSL Error</translation>
    </message>
    <message>
        <source>NoError:Description</source>
        <translation>Unknown SSL error occurred</translation>
    </message>
    <message>
        <source>UnableToGetIssuerCertificate:Name</source>
        <translation>Unable To Get Issuer Certificate Error</translation>
    </message>
    <message>
        <source>UnableToGetIssuerCertificate:Description</source>
        <translation>Client was not able to get a issuer certificate</translation>
    </message>
    <message>
        <source>UnableToDecryptCertificateSignature:Name</source>
        <translation>Unable To Decrypt Certificate Signature Error</translation>
    </message>
    <message>
        <source>UnableToDecryptCertificateSignature:Description</source>
        <translation>Client was not able to decrypt the certificate signature</translation>
    </message>
    <message>
        <source>UnableToDecodeIssuerPublicKey:Name</source>
        <translation>Unable To Decode Issuer Public Key Error</translation>
    </message>
    <message>
        <source>UnableToDecodeIssuerPublicKey:Description</source>
        <translation>Client was not able to decode the issuer public key</translation>
    </message>
    <message>
        <source>CertificateSignatureFailed:Name</source>
        <translation>Certificate Signature Failed Error</translation>
    </message>
    <message>
        <source>CertificateSignatureFailed:Description</source>
        <translation>Client was not able to sign the certificate</translation>
    </message>
    <message>
        <source>CertificateNotYetValid:Name</source>
        <translation>Certificate Not Yet Valid Error</translation>
    </message>
    <message>
        <source>CertificateNotYetValid:Description</source>
        <translation>Client was not able to get a valid certificate</translation>
    </message>
    <message>
        <source>CertificateExpired:Name</source>
        <translation>Certificate Expired Error</translation>
    </message>
    <message>
        <source>CertificateExpired:Description</source>
        <translation>The certificate is expired</translation>
    </message>
    <message>
        <source>InvalidNotBeforeField:Name</source>
        <translation>Invalid Not Before Field Error</translation>
    </message>
    <message>
        <source>InvalidNotBeforeField:Description</source>
        <translation>Client obtained a not valid certificate</translation>
    </message>
    <message>
        <source>InvalidNotAfterField:Name</source>
        <translation>Client obtained a not valid certificate</translation>
    </message>
    <message>
        <source>InvalidNotAfterField:Description</source>
        <translation>Client obtained a not valid certificate</translation>
    </message>
    <message>
        <source>SelfSignedCertificate:Name</source>
        <translation>Self Signed Certificate Error</translation>
    </message>
    <message>
        <source>SelfSignedCertificate:Description</source>
        <translation>Client could not self sign the certificate</translation>
    </message>
    <message>
        <source>SelfSignedCertificateInChain:Name</source>
        <translation>Self Signed Certificate In Chain Error</translation>
    </message>
    <message>
        <source>SelfSignedCertificateInChain:Description</source>
        <translation>Client could not self sign the certificate in chain</translation>
    </message>
    <message>
        <source>UnableToGetLocalIssuerCertificate:Name</source>
        <translation>Unable To Get Local Issuer Certificate Error</translation>
    </message>
    <message>
        <source>UnableToGetLocalIssuerCertificate:Description</source>
        <translation>Client was not able to get a local issuer for the certificate</translation>
    </message>
    <message>
        <source>UnableToVerifyFirstCertificate:Name</source>
        <translation>Unable To Verify First Certificate Error</translation>
    </message>
    <message>
        <source>UnableToVerifyFirstCertificate:Description</source>
        <translation>Client was not able to verify the first certificate</translation>
    </message>
    <message>
        <source>CertificateRevoked:Name</source>
        <translation>Certificate Revoked Error</translation>
    </message>
    <message>
        <source>CertificateRevoked:Description</source>
        <translation>The certificate was revoked</translation>
    </message>
    <message>
        <source>InvalidCaCertificate:Name</source>
        <translation>Invalid Ca Certificate Error</translation>
    </message>
    <message>
        <source>InvalidCaCertificate:Description</source>
        <translation>The CA certificate is not valid</translation>
    </message>
    <message>
        <source>PathLengthExceeded:Name</source>
        <translation>Path Length Exceeded Error</translation>
    </message>
    <message>
        <source>PathLengthExceeded:Description</source>
        <translation>The length of the path exceeded the limit</translation>
    </message>
    <message>
        <source>InvalidPurpose:Name</source>
        <translation>Invalid Purpose Error</translation>
    </message>
    <message>
        <source>InvalidPurpose:Description</source>
        <translation>The purpose of the certificate is invalid</translation>
    </message>
    <message>
        <source>CertificateUntrusted:Name</source>
        <translation>Certificate Untrusted Error</translation>
    </message>
    <message>
        <source>CertificateUntrusted:Description</source>
        <translation>The certificate is not trusted</translation>
    </message>
    <message>
        <source>CertificateRejected:Name</source>
        <translation>Certificate Rejected Error</translation>
    </message>
    <message>
        <source>CertificateRejected:Description</source>
        <translation>The certificate is rejected</translation>
    </message>
    <message>
        <source>SubjectIssuerMismatch:Name</source>
        <translation>Subject Issuer Mismatch Error</translation>
    </message>
    <message>
        <source>SubjectIssuerMismatch:Description</source>
        <translation>There was a mismatch between the subject and the issuer</translation>
    </message>
    <message>
        <source>AuthorityIssuerSerialNumberMismatch:Name</source>
        <translation>Authority Issuer Serial Number Mismatch Error</translation>
    </message>
    <message>
        <source>AuthorityIssuerSerialNumberMismatch:Description</source>
        <translation>There was a mismatch in the serial number of the authority issuer</translation>
    </message>
    <message>
        <source>NoPeerCertificate:Name</source>
        <translation>No Peer Certificate Error</translation>
    </message>
    <message>
        <source>NoPeerCertificate:Description</source>
        <translation>The peer certificate was missing</translation>
    </message>
    <message>
        <source>HostNameMismatch:Name</source>
        <translation>Host Name Mismatch Error</translation>
    </message>
    <message>
        <source>HostNameMismatch:Description</source>
        <translation>The host name was not valid</translation>
    </message>
    <message>
        <source>NoSslSupport:Name</source>
        <translation>No SSL Support Error</translation>
    </message>
    <message>
        <source>NoSslSupport:Description</source>
        <translation>The host does not support SSL</translation>
    </message>
    <message>
        <source>CertificateBlacklisted:Name</source>
        <translation>Certificate Blacklisted Error</translation>
    </message>
    <message>
        <source>CertificateBlacklisted:Description</source>
        <translation>The certificate is in the black list</translation>
    </message>
    <message>
        <source>UnspecifiedError:Name</source>
        <translation>Unspecified Error</translation>
    </message>
    <message>
        <source>UnspecifiedError:Description</source>
        <translation>An unspecified SSL error has occurr
ed</translation>
    </message>
    <message>
        <source>UnknownError:Name</source>
        <translation>Unknown SSL Network Error</translation>
    </message>
    <message>
        <source>UnknownError:Description</source>
        <translation>An unknown SSL network error has occurred</translation>
    </message>
</context>
<context>
    <name>JSONError</name>
    <message>
        <source>NoError:Name</source>
        <translation>Unknown JSON Error</translation>
    </message>
    <message>
        <source>NoError:Description</source>
        <translation>Unknown error occurred</translation>
    </message>
    <message>
        <source>UnterminatedObject:Name</source>
        <translation>Unterminated Object Error</translation>
    </message>
    <message>
        <source>UnterminatedObject:Description</source>
        <translation>An object is not correctly terminated with a closing curly bracket</translation>
    </message>
    <message>
        <source>MissingNameSeparator:Name</source>
        <translation>Missing Name Separator Error</translation>
    </message>
    <message>
        <source>MissingNameSeparator:Description</source>
        <translation>A comma separating different items is missing</translation>
    </message>
    <message>
        <source>UnterminatedArray:Name</source>
        <translation>Unterminated Array Error</translation>
    </message>
    <message>
        <source>UnterminatedArray:Description</source>
        <translation>The array is not correctly terminated with a closing square bracket</translation>
    </message>
    <message>
        <source>MissingValueSeparator:Name</source>
        <translation>Missing Value Separator Error</translation>
    </message>
    <message>
        <source>MissingValueSeparator:Description</source>
        <translation>A colon separating keys from values inside objects is missing</translation>
    </message>
    <message>
        <source>IllegalValue:Name</source>
        <translation>Illegal Value Error</translation>
    </message>
    <message>
        <source>IllegalValue:Description</source>
        <translation>The value is illegal</translation>
    </message>
    <message>
        <source>TerminationByNumber:Name</source>
        <translation>Termination By Number Error</translation>
    </message>
    <message>
        <source>TerminationByNumber:Description</source>
        <translation>The input stream ended while parsing a number</translation>
    </message>
    <message>
        <source>IllegalNumber:Name</source>
        <translation>Illegal Number Error</translation>
    </message>
    <message>
        <source>IllegalNumber:Description</source>
        <translation>The number is not well formed</translation>
    </message>
    <message>
        <source>IllegalEscapeSequence:Name</source>
        <translation>Illegal Escape Sequence Error</translation>
    </message>
    <message>
        <source>IllegalEscapeSequence:Description</source>
        <translation>An illegal escape sequence occurred in the input</translation>
    </message>
    <message>
        <source>IllegalUTF8String:Name</source>
        <translation>Illegal UTF8 String Error</translation>
    </message>
    <message>
        <source>IllegalUTF8String:Description</source>
        <translation>An illegal UTF8 sequence occurred in the input</translation>
    </message>
    <message>
        <source>UnterminatedString:Name</source>
        <translation>Unterminated String Error</translation>
    </message>
    <message>
        <source>UnterminatedString:Description</source>
        <translation>A string wasn&apos;t terminated with a quote</translation>
    </message>
    <message>
        <source>MissingObject:Name</source>
        <translation>Missing Object Error</translation>
    </message>
    <message>
        <source>MissingObject:Description</source>
        <translation>An object was expected but could not be found</translation>
    </message>
    <message>
        <source>DeepNesting:Name</source>
        <translation>Deep Nesting Error</translation>
    </message>
    <message>
        <source>DeepNesting:Description</source>
        <translation>The JSON document is too deeply nested for the parser to parse it</translation>
    </message>
    <message>
        <source>DocumentTooLarge:Name</source>
        <translation>Document Too Large Error</translation>
    </message>
    <message>
        <source>DocumentTooLarge:Description</source>
        <translation>The JSON document is too large to be parsed</translation>
    </message>
    <message>
        <source>UnknownError:Name</source>
        <translation>Unknown JSON Error</translation>
    </message>
    <message>
        <source>UnknownError:Description</source>
        <translation>An unknown JSON error has occurred</translation>
    </message>
</context>
</TS>
